CMakeLists.txt

```diff
@@ -183,24 +183,31 @@ endif ()
 # Make sure the final executable has symbols exported
 set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -rdynamic")
 
-if (OS_LINUX)
-    find_program (OBJCOPY_PATH NAMES "llvm-objcopy" "llvm-objcopy-12" "llvm-objcopy-11" "llvm-objcopy-10" "llvm-objcopy-9" "llvm-objcopy-8" "objcopy")
-    if (OBJCOPY_PATH)
-        message(STATUS "Using objcopy: ${OBJCOPY_PATH}.")
-
+find_program (OBJCOPY_PATH NAMES "llvm-objcopy" "llvm-objcopy-12" "llvm-objcopy-11" "llvm-objcopy-10" "llvm-objcopy-9" "llvm-objcopy-8" "objcopy")
+if (OBJCOPY_PATH)
+   message(STATUS "Using objcopy: ${OBJCOPY_PATH}.")
+   if (OS_LINUX)
```

Comment:
I left this chunk setting `OBJCOPY_ARCH_OPTIONS`. It's no longer used as far as I can tell, but I wasn't sure if there were plans to use it later. Let me know if you'd like to remove this section in this PR as well.

Reply:
We have no plans to reuse this option, you can remove it freely.